ethereumj-core/src/main/java/org/ethereum/core/Transaction.java

```diff
@@ -374,7 +374,7 @@ public boolean isContractCreation() {
 
     public ECKey getKey() {
         byte[] hash = getRawHash();
-        return ECKey.recoverFromSignature(signature.v, signature, hash);
+        return ECKey.recoverFromSignature(0, signature, hash);
```

Comment:
Use `0` as `recId` will cause bugs. Take a look at [`ECKey.signatureToKeyBytes`](https://github.com/ethereum/ethereumj/blob/develop/ethereumj-core/src/main/java/org/ethereum/crypto/ECKey.java#L826) to get an idea of how `recId` is decoded from signature.

Reply:
use ECKey.signatureToKey(hash, getSignature())